src/bin/helpers.js

```diff
@@ -27,6 +27,16 @@ function displayNonInitializedFolderErrorMessage() {
     );
 }
 
+// Check if harmonic.json file exists
+function isHarmonicProject(sitePath) {
```

Comment:
Looks like this function was moved from below `findHarmonicRoot` to here. It would be nice to avoid unnecessarily moving code around as to reduce the diff and make history/blame easier to follow.